server/app.js

```diff
@@ -3,14 +3,23 @@ const express = require("express");
 const { join } = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
+const mongoose = require("mongoose");
+require('dotenv').config({ path: './.env'})
 
 const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
+const signupRouter = require("./routes/signup");
 
 const { json, urlencoded } = express;
 
 var app = express();
 
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+
+const connectionString = 'mongodb+srv://' + process.env.REACT_APP_MONGO_ID + ':' + process.env.REACT_APP_MONGO_PW + '@cluster0.q0hmz.mongodb.net/project?retryWrites=true&w=majority'
```

Comment:
This can be moved fully to the environment file as (MONGO_DB_URI) instead of two separate for ID and password

Reply:
This is resolved in the BE branch